MechPredict.pm

```diff
@@ -0,0 +1,288 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute  
+Copyright [2016-2024] EMBL-European Bioinformatics Institute  
+
+Licensed under the Apache License, Version 2.0 (the "License");  
+you may not use this file except in compliance with the License.  
+You may obtain a copy of the License at  
+
+     http://www.apache.org/licenses/LICENSE-2.0  
+
+Unless required by applicable law or agreed to in writing, software  
+distributed under the License is distributed on an "AS IS" BASIS,  
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  
+See the License for the specific language governing permissions and  
+limitations under the License.  
+
+=head1 CONTACT  
+
+Ensembl <http://www.ensembl.org/info/about/contact/index.html>  
+
+=cut  
+
+=head1 NAME  
+
+MechPredict  
+
+=head1 SYNOPSIS  
+
+ mv MechPredict.pm ~/.vep/Plugins  
+ ./vep -i input.vcf --plugin MechPredict,file=mechpredict_data.tsv  
+
+=head1 DESCRIPTION  
+
+This is a plugin for the Ensembl Variant Effect Predictor (VEP) that annotates missense variants with predicted 
+dominant-negative (DN), gain-of-function (GOF), or loss-of-function (LOF) mechanisms derived from a
+Support Vector Classification (SVC) model (Badonyi et al., 2024).
+
```

Comment:
We should add a further comment for clarity: "These probabilities do not predict a gene is involved in disease, they predict the most likely molecular mechanism by which deleterious variants in a given gene could cause disease, if it was a dominant disease gene.